system-monitor@paradoxxx.zero.gmail.com/prefs.js

```diff
@@ -66,12 +69,49 @@ function check_sensors(sensor_type) {
     return [sensor_list, string_list];
 }
 
+function box(isHorizontal, spacing, hasBorder) {
+    const options = { };
+
+    if (typeof spacing !== 'undefined') {
+        options.spacing = spacing;
+    }
+
+    if (shellMajorVersion < 40) {
+        if (hasBorder) {
+            options.border_width = 10;
+        }
+
+        return isHorizontal ? new Gtk.HBox(options) : new Gtk.VBox(options);
+    }
+
+    if (hasBorder) {
+        options.margin_top = 10;
+        options.margin_bottom = 10;
+        options.margin_start = 10;
+        options.margin_end = 10;
+    }
+
+    options.orientation = isHorizontal ?
+        Gtk.Orientation.HORIZONTAL : Gtk.Orientation.VERTICAL;
+
+    const aliasBox = new Gtk.Box(options);
+
+    if (isHorizontal) {
+        aliasBox.set_homogeneous(true);
+    }
+
+    aliasBox.add = aliasBox.append;
+    aliasBox.pack_start = aliasBox.prepend;
+    aliasBox.pack_end = aliasBox.prepend;
+
+    return aliasBox;
+}
 
 const ColorSelect = class SystemMonitor_ColorSelect {
     constructor(name) {
         this.label = new Gtk.Label({label: name + _(':')});
         this.picker = new Gtk.ColorButton();
-        this.actor = new Gtk.HBox({spacing: 5});
+        this.actor = box(5);
```

Comment:
this one should have been box(true, 5) probably!

Reply:
Thanks for that catch!